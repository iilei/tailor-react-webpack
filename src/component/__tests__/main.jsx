/**
 * Created by iilei on 29.01.17.
 */

/* eslint-disable */

it('should render', () => {
  const checkbox = shallow(
    <Main labelOn="On" labelOff="Off" />
  );

  checkbox.find('input').simulate('change');

  expect(checkbox).toMatchSnapshot();
});


import Main from '../main';

it('Main changes the text after click', () => {
  // Render a checkbox with label in the document
  const checkbox = shallow(
    <Main labelOn="On" labelOff="Off" />
  );

  expect(checkbox.text()).toEqual('Off');

  checkbox.find('input').simulate('change');

  expect(checkbox.text()).toEqual('On');
});
