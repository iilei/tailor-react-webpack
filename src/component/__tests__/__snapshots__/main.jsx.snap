exports[`test should render 1`] = `
<label
  htmlFor="foo">
  <input
    checked={true}
    onChange={[Function]}
    type="checkbox" />
  On
</label>
`;
